ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-vp9-with-svt-hevc-av1.patch

```diff
@@ -632,6 +633,177 @@ index eb70421c44..0a7f85e0e9 100644
      if (st->codecpar->codec_id == AV_CODEC_ID_VP9)
          ret = ff_stream_add_bitstream_filter(st, "vp9_superframe", NULL);
      else if (st->codecpar->codec_id == AV_CODEC_ID_AV1)
+diff --git a/libavformat/matroskaenc.c b/libavformat/matroskaenc.c
+index cef504fa05..fc652db027 100644
+--- a/libavformat/matroskaenc.c
++++ b/libavformat/matroskaenc.c
+@@ -161,6 +161,9 @@ typedef struct MatroskaMuxContext {
+     int64_t *stream_duration_offsets;
+ 
+     int allow_raw_vfw;
++
++    int simple_block_timecode;
++    int accumulated_cluster_timecode;
+ } MatroskaMuxContext;
+ 
+ /** 2 bytes * 7 for EBML IDs, 7 1-byte EBML lengths, 6 1-byte uint,
+@@ -2180,7 +2183,12 @@ static void mkv_write_block(AVFormatContext *s, AVIOContext *pb,
+     put_ebml_num(pb, size + 4, 0);
+     // this assumes stream_index is less than 126
+     avio_w8(pb, 0x80 | track_number);
+-    avio_wb16(pb, ts - mkv->cluster_pts);
++
++    if ((par->codec_type == AVMEDIA_TYPE_VIDEO) && (par->codec_id == AV_CODEC_ID_VP9))
++        avio_wb16(pb, mkv->simple_block_timecode);
```

Comment:
Will this patch break libvpx?

Reply:
Yes, it will.

Reply:
Can we add some check and only run the logic for svt-vp9?